addons/stock_account/models/product.py

```diff
@@ -695,13 +695,29 @@ def _svl_replenish_stock(self, description, products_orig_quantity_svl):
             ('lot_id', '!=', False),
             ], ['product_id', 'location_id', 'lot_id'], ['quantity:sum'],
             having=[('quantity:sum', '>', 0)])
+        no_lot_quants = self.env['stock.quant']._read_group([
+            ('product_id', 'in', self.product_variant_ids.filtered("lot_valuated").ids),
```

Comment:
```suggestion
            ('product_id', 'in', self.filtered("lot_valuated").ids),
```
self is already a `product.product`

Reply:
Yes, I copy/pasted the read_groups above.
I thought that there was a reason for using the product variants, so I double-checked that it worked as expected and left it here :shrug: